### PR TITLE
Shakil

### PR DESCRIPTION
**Pull request by mdshakilahmad**
header and hero complete.

### CODE REVIEW

File: layout/Header.tsx [hunk 1]
```diff
@@ -6,38 +6,35 @@ export default function Header() {
...
-          <Link href="/" className="text-3xl font-semibold">
-            Sasrio
+          <Link href="/" className="">
+            <img src="/images/logo.png" alt="logo" />
           </Link>
         </div>
-        <ul className="flex gap-5 font-normal text-lg">
-          <li>
+        <ul className="flex gap-5 font-medium text-lg font-manrope">
+          <li className="dot">
```
Comment (devrashed01) [inline layout/Header.tsx:14]: Don't need to repeat same class. Rather use one class ul then write css on css file for li

File: package.json [hunk 2]
```diff
@@ -12,7 +12,8 @@
     "@fortawesome/fontawesome-svg-core": "^6.4.2",
     "next": "latest",
     "react": "latest",
-    "react-dom": "latest"
+    "react-dom": "latest",
+    "react-icons": "^4.11.0"
```
Comment (devrashed01) [inline package.json:16]: Remove ^

**Review by devrashed01 — CHANGES_REQUESTED** [review]

File: layout/Header.tsx (hunk 1 above)
Comment (mdshakilahmad) [reply layout/Header.tsx]: solved

File: package.json (hunk 2 above)
Comment (mdshakilahmad) [reply package.json]: fixed

**Review by devrashed01 — APPROVED** [review]

**Pull request merged**